src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java

```diff
@@ -1149,7 +1158,7 @@ private AdsDataProvider() {
   }
 
   @DataProvider
-  public static Object[][] adsRecoveryOasisHopToTaboola() {
+  public static Object[][] AdsRecoverySourcePointOasisHopToTaboola() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS00100)